render/respond.go

```diff
@@ -0,0 +1,111 @@
+package render
+
+import (
+	"bytes"
+	"encoding/json"
+	"encoding/xml"
+	"net/http"
+	"reflect"
+
+	"golang.org/x/net/context"
+)
+
+var Respond = DefaultRespond
+
+func DefaultRespond(ctx context.Context, w http.ResponseWriter, v interface{}) {
```

Comment:
```go
w.Header().Set("Header", "first value")
w.Header().Set("Header", "no, sorry, this is the right value")
```
equals
`w.Header().Set("Header", []string{"first value", "no, sorry, this is the right value")}`
and not
`w.Header().Set("Header", "no, sorry, this is the right value")`